frontend/config-core_demo.json

```diff
@@ -35,5 +35,11 @@
   },
   "@openmrs/esm-styleguide": {
     "Brand color #1": "#005d5d"
+  },
+  "@openmrs/esm-billing-app": {
+    "serviceTypes": {
+      "billableService": "0dfe7d9f-1dc9-4e68-b4cb-d19e1a006c54"
+    },
+    "defaultCurrency": "UGX"
```

Comment:
Is this supposed to be UG. The Pr looks good

Reply:
Oh I was just testing... we could use USD since its more neutral and universally used